File: src/main/java/no/nav/syfo/rest/domain/RSNaermesteLeder.java
package no.nav.syfo.rest.domain;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;

import java.time.LocalDate;
import java.time.LocalDateTime;


@Data
@Accessors(fluent = true)
@EqualsAndHashCode
public class RSNaermesteLeder extends RSPerson {
    public String virksomhetsnummer;
    public boolean erAktiv;
    public LocalDate aktivFom;
    public LocalDate aktivTom;
    public String navn = " ";
    public String fnr;
    public String epost;
    public String tlf;
    public LocalDateTime sistInnlogget;
    public Boolean samtykke;
    public RSEvaluering evaluering;
}

File: src/main/kotlin/no/nav/syfo/narmesteleder/controller/RSNaermesteLeder.kt
package no.nav.syfo.narmesteleder.controller

import no.nav.syfo.rest.domain.RSEvaluering
import java.time.LocalDate
import java.time.LocalDateTime

data class RSNaermesteLeder(
        val virksomhetsnummer: String,
        val erAktiv: Boolean,
        val aktivFom: LocalDate,
        val aktivTom: LocalDate?,
        val navn: String = " ",
        val fnr: String,
        val epost: String?,
        val tlf: String?,
        val sistInnlogget: LocalDateTime?,
        val samtykke: Boolean?,
        val evaluering: RSEvaluering?
)
